src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.WinRT.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
```

Comment:
> I just assume the WinRT code is going away in 2.1.

That is (most likely) correct. I have a meeting regarding this specific dependency next week. Current plans _should_ remove the need for a WinRT specific build.